src/Tribe/Status/Abstract_Commerce.php

```diff
@@ -183,29 +182,42 @@ public function get_name_and_sold_for_ticket( $ticket_sold, $post_id ) {
 		echo esc_html( $sold_message );
 
 		return ob_get_clean();
-
 	}
 
 	/**
-	 * Get the Available and Incomplete Counts for a Ticket
+	 * Get the available and incomplete counts for a Ticket.
 	 *
 	 * @since 4.10
 	 *
-	 * @param $ticket_sold object an object of the ticket to get counts
+	 * @param array $ticket_sold Ticket array to get counts.
 	 *
-	 * @return bool|string a string of available and/or incomplete counts for a ticket
+	 * @return string The available and/or incomplete counts for a ticket.
 	 */
 	public function get_available_incomplete_counts_for_ticket( $ticket_sold ) {
-
-		$availability = array();
-		if (  $ticket_sold['ticket']->available() > 0 ) {
-			$availability['available'] = sprintf( '%s %s%s',
-				esc_html( $ticket_sold['ticket']->available() ),
-				esc_html__( 'available', 'event-tickets' ),
-				$this->get_availability_by_ticket_tooltip( $ticket_sold )
-			);
+		/** @var Tribe__Tickets__Ticket_Object $ticket_object */
```

Comment:
:no_entry_sign: Missing short description in doc comment

<sub>hash: 69902ae43aaa9634389ebdd2bfae58df</sub>